### PR TITLE
Added Dockerfile&.zshrc for Zsh

### PR DESCRIPTION
**Pull request by AkihikoITOH**

### CODE REVIEW

File: zsh/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:wily
+
+RUN apt-get update -y && apt-get install -y zsh
```
Comment (jfrazelle) [inline zsh/Dockerfile]: can you make this like the others with rm -rf /var/apt/lists...

File: zsh/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:wily
```
Comment (jfrazelle) [inline zsh/Dockerfile]: this should be a debian distro ;)

**Comment by AkihikoITOH** [comment]
@jfrazelle I have changed base image to 'debian:jessie' and added `rm -rf /var/apt/lists/*` following your advice. Thank you.

**Comment by jfrazelle** [comment]
thanks!

**Pull request merged**